docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -203,6 +203,11 @@ For security reasons, commits by users outside of your organization will
 not automatically be run. A patch will still be created and must be
 manually authorized to run by a logged-in user.
 
+#### Limiting when PR patches will run
```

Comment:
can't check the ticket description / comments rn since Jira is being chaotic... but one thing I can think of with implementing the feature this way is that the oldest allowed merge base will get outdated very easily (sorry I missed the PR where the functionality was actually introduced). Do we want this to be a number of days instead, so it stays up to date, or is it meant more to prevent specific bad commits from being included kind of? (Fine with whatever, just wondering if we considered that)

Reply:
The field won't get outdated since its purpose is to reject any PRs that come in that have a merge base before the "oldest allowed merge base". From the ticket description and the ask, it seems setting an explicit SHA to not allow merge bases before is the desired behavior.